doctrine.rst

```diff
@@ -73,7 +73,7 @@ database for you:
 
 .. code-block:: terminal
 
-    $ php bin/console doctrine:database:create
+    $ php bin/console doctrine:database:create db_name
```

Comment:
The `doctrine:database:create` command doesn't have arguments. What makes you think this change is necessary?